test/functional/wallet_migration.py

```diff
@@ -585,7 +585,8 @@ def test_direct_file(self):
         )
         assert (self.master_node.wallets_path / "plainfile").is_file()
 
-        self.master_node.migratewallet("plainfile")
+        migrate_res = self.master_node.migratewallet("plainfile")
+        assert_equal(f"plainfile_{int(time.time())}.legacy.bak", os.path.basename(migrate_res["backup_path"]))
```

Comment:
Maybe you can use `mocktime` here for consistency?

```diff --git a/test/functional/wallet_migration.py b/test/functional/wallet_migration.py
--- a/test/functional/wallet_migration.py
+++ b/test/functional/wallet_migration.py
@@ -585,8 +585,10 @@ class WalletMigrationTest(BitcoinTestFramework):
         )
         assert (self.master_node.wallets_path / "plainfile").is_file()
 
+        mocked_time = int(time.time())
+        self.master_node.setmocktime(mocked_time)
         migrate_res = self.master_node.migratewallet("plainfile")
-        assert_equal(f"plainfile_{int(time.time())}.legacy.bak", os.path.basename(migrate_res["backup_path"]))
+        assert_equal(f"plainfile_{mocked_time}.legacy.bak", os.path.basename(migrate_res["backup_path"]))

```

Reply:
Thank you, fixed!